File: assignment_1/Product.java
// This class stores details of any product in the store.
public class Product {
    int p_id;
    String product_name;
    String brand;
    double Price;
    String Category;
    String Warranty;
    int stock;

    // Prints all product details
    void Display() {
        System.out.println("Product ID    : " + p_id);
        System.out.println("Product Name  : " + product_name);
        System.out.println("Brand         : " + brand);
        System.out.println("Price         : Rs." + Price);
        System.out.println("Category      : " + Category);
        System.out.println("Warranty      : " + Warranty);
        System.out.println("Stock         : " + stock);
    }
}

File: python/product.py
class Product:
    def __init__(self):
        self.p_id = 0
        self.product_name = ""
        self.brand = ""
        self.Price = 0.0
        self.Category = ""
        self.Warranty = ""
        self.stock = 0

    def Display(self):
        print("Product ID    :", self.p_id)
        print("Product Name  :", self.product_name)
        print("Brand         :", self.brand)
        print("Price         : Rs.", self.Price)
        print("Category      :", self.Category)
        print("Warranty      :", self.Warranty)
        print("Stock         :", self.stock)